runbot_send_email/README.rst

```diff
@@ -0,0 +1,33 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Runbot Send Email Result
+========================
+
+This module sends the results of the RunBot builds through email.
```

Comment:
Please set this help a little deeper, showng the use case with followerd and with PS of the resulting email sent (once finished).

On this way the new fields which will be used to build the links will be better supported.